packages/core/core.ts

```diff
@@ -135,11 +137,10 @@ function cleanStack(stack: string): string {
 	return result.join("\n");
 }
 
-function importModule(name: string) {
-	// eslint-disable-next-line @typescript-eslint/no-var-requires
-	return require(name);
+async function importModule(name: string) {
+	return await import(name);
```

Comment:
The `await` could be removed, as the function already returns a promise due to the `async` keyword.

Reply:
Done